model/mod_list.py

```diff
@@ -134,16 +251,18 @@ def handle_rows_inserted(self, parent: QModelIndex, first: int, last: int) -> No
                 widget = ModListItemInner(
                     data, self.width(), self.steam_icon_path, self.ludeon_icon_path
                 )
+                if data.get("invalid"):
```

Comment:
Won't this overwrite any other style that was already set for the QLabel? It would be better to define a "valid" style and an "invalid" style and switch object names.

Reply:
Possibly, but i don't think that we are currently doing anything that it is overwriting. The original style is also set otherwise (so if people don't have invalid mods, that wouldn't apply anyways)

This can be considered in future UI redesign/reimplementation tasks